### PR TITLE
init_project

### PR DESCRIPTION
**Pull request by dungdnv-1036**
init_project

### CODE REVIEW

**Comment by dungdnv-1036** [comment]
Screenshot from 2020-08-26 13-33-37

**Comment by dungdnv-1036** [comment]
ready

File: .framgia-ci-reports/rubocop.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version='1.0'?>
```
Comment (saobv-2223) [inline .framgia-ci-reports/rubocop.xml]: image

File: .framgia-ci-result.temp.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+rails_best_practices:
```
Comment (saobv-2223) [inline .framgia-ci-result.temp.yml]: image

File: app/javascript/packs/application.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
+require("channels")
```
Comment (saobv-2223) [inline app/javascript/packs/application.js]: dùng '
xóa hết comment đi bạn

File: config/credentials.yml.enc [hunk 4]
```diff
@@ -0,0 +1 @@
+TzII0SyXs0QQcP7UPb4jFEKKgvw4/cCsb1BJ09iJp5SOO5nGtgOQ1cG8AmH4umpa3izghR9vsxS25Kivsupl3jhldTX3XoaAy1XdzjK5TdY0aVK/6K5Jwyi/ReNOHNePJ8CmCMZepn6tN9T5xC5OtA5gcK9Ge7MhY+hb1L5lb3CdGSNREXG6M/jWn7lQoo725C3ESNxNJajNV8SUcoXWDv1qz5DIVVdcYuoaSSj8k32uul2c71U2ESf+A4PyKbqDqbROoUxkg/wXXPehGdDkWcarkAy9/KoNTBemJOrwf4yhULNgJ+Yyd8lW+UnoRyVMNYvtOdmMVHd1opahM/fjaRW/06K1kl5sDuGSsfKNOs6XrLOd2sAhoxMrFkPGEmPbpxWP8Hop3MGpVLl2ab5ZErVVhYXJiLKKkm67--NNBkwcqk/zfi1Ogd--2GrjsK+DaPCxgTjbHJ6KGw==
```
Comment (saobv-2223) [inline config/credentials.yml.enc]: image

**Comment by dungdnv-1036** [comment]
https://edu-redmine.sun-asterisk.vn/issues/28103

**Comment by dungdnv-1036** [comment]
ready

**Pull request merged**